packages/vue/src/components/molecules/SfGallery/SfGallery.js

```diff
@@ -41,6 +46,18 @@ export default {
     };
   },
 
+  computed: {
+    mapPictures() {
+      // map images to handle picture tags with SfImage
+      return this.images.map(({ normal, big }) => {
```

Comment:
@leomp12 something is wrong on mobile devices please fix it 
![_](https://user-images.githubusercontent.com/12138170/65506371-b3eea680-decb-11e9-8c76-c5c4cbec6342.png)

Reply:
I think the example on storybook has only `big` size, I'll check the `normal` size and if not present render a normal `<img>` instead of `<picture>`...

Reply:
@przspa done :)